test/functional/legacy/062_tab_pages_spec.lua

```diff
@@ -0,0 +1,272 @@
+-- Tests for tab pages
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source, clear, execute, expect, eval, eq =
+  helpers.feed, helpers.insert, helpers.source, helpers.clear,
+  helpers.execute, helpers.expect, helpers.eval, helpers.eq
+
+describe('tab pages', function()
+  setup(clear)
+
+  it('is working', function()
+    insert('Results:')
+
+    execute('lang C')
```

Comment:
It doesn't fail for me but from `:h :lang` I infer that it is needed for the test of `E474` in this test file. Maybe someone with a non English system or more knowledge can say more.